lib/flipper/ui.rb

```diff
@@ -44,6 +44,8 @@ def self.app(flipper, options = {})
       builder.use Flipper::Middleware::Memoizer, flipper
       builder.use Middleware, flipper
       builder.run app
+      klass = self
+      builder.define_singleton_method(:inspect) { klass } # pretty rake routes output
```

Comment:
rake routes calls `inspect`.  Line 47 just keeps a reference to the class `Flipper::UI` in case for some reason it changes in the future will still reference correct class and not just hard code 'Flipper::UI', then we just override inspect on the builder instance, which is the object rake routes sends the `inspect` message to